src/styles/main.scss

```diff
@@ -1 +1,83 @@
 // put code here
+
+$time: 45;
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  margin: 0;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 500px;
+  height: 500px;
+  border: 5px dotted black;
+}
+
+.stopwatch__center {
+  width: 30px;
+  height: 30px;
+  background-color: orange;
+}
+
+.stopwatch,
+.stopwatch__center {
+  border-radius: 50%;
+}
+
+.stopwatch__arrow {
+  position: absolute;
+  bottom: 50%;
+  transform-origin: bottom;
+}
+
+.stopwatch__arrow--big {
+  height: 250px;
+  width: 8px;
+  background-color: green;
+  z-index: -1;
+
+  animation-name: big-arrow-move;
+  animation-duration: #{$time}s;
+  animation-timing-function: steps($time, end);
+  animation-fill-mode: forwards;
+}
+
+.stopwatch__arrow--small {
+  height: 125px;
+  width: 12px;
+  background-color: blue;
+  z-index: -2;
+
+  animation-name: small-arrow-move;
+  animation-duration: #{$time}s;
+  animation-timing-function: linear;
+  animation-fill-mode: forwards;
+}
+
+@keyframes big-arrow-move {
+  from {
+    transform: rotate(0);
+  }
+
+  to {
+    transform: rotate(#{$time / 60 * 360}deg);
+  }
+}
+
+@keyframes small-arrow-move {
+  from {
+    transform: rotate(0);
+  }
+
+  to {
+    transform: rotate(#{$time / 3600 * 360}deg);
+  }
+}
```

Comment:
you dont need 2 keyframes for same circle rotation, and now its finite but must be not